lib/index.js

```diff
@@ -7,7 +7,13 @@ var load = require('./load')
 /**
  * Load audio
  */
-module.exports = function (path, options) {
+module.exports = function (path, options, cb) {
+  if (options instanceof Function) {
+    cb = options
+    options = {}
+  }
+  options = options || {}
+  options.ready = cb || function () {}
```

Comment:
This potentially changes the received options object, causing a side effect. I would prefer not to do it. For me, a better solution would be:

```js
options = options || {}
if (typeof options === 'function') {
  options = { ready: options }
}
````

But I'm open (as always) to suggestions.